src/views/TodoList.jsx

```diff
@@ -20,10 +22,24 @@ function TodoList() {
     [todoList]
   );
 
+  const fitterTodoList = todoList.filter((task) => {
```

Comment:
The variable name should be `filteredTodos`. This calculation should be wrapped by useMemo to reduce the performance cost